docs/draft-hfc-proposal.md

```diff
@@ -0,0 +1,70 @@
+The options for how we see to proceed.
+
+1) retain the status quo
+
+2) upstream block counting into the Ledger governance actions related to eras
+
+3) remove block counting, but keep double stability (ie permits the HFC interpreter to say Just Nothing Just)
+
+    3a) do not add HLQ mitigations
+    3b) add HLQ mitigations
+    3c) more exotic/subtle: change block counting to be a disjunct instead of a conjunct prevents Just -> Nothing (since that only arises when switching to a denser chain)
+
+4) remove block counting, remove double stability, and add HLQ mitigations
```

Comment:
What about adding a few examples for illustration, ie what would be the result of doing time translations/forecasts in different areas of an epoch when there is (no) era transition.